apps/journeys-admin/src/components/Editor/Slider/Settings/CanvasDetails/AddBlock/AddBlock.spec.tsx

```diff
@@ -130,14 +135,13 @@ describe('AddBlock', () => {
       </MockedProvider>
     )
 
-    expect(screen.getByText('activeSlide: 1')).toBeInTheDocument()
     await waitFor(() =>
```

Comment:
There's aren't necessarily your changes, but if we can also clean some of the extra stuff that's not needed while we're here.

```suggestion
    const closeButton = screen.getByTestId('X2Icon')
    
    expect(closeButton).toBeInTheDocument()
    
    await waitFor(() => userEvent.click(closeButton))
```